src/pymor/algorithms/gram_schmidt.py

```diff
@@ -65,7 +65,7 @@ def gram_schmidt(A, product=None, return_R=False, atol=1e-13, rtol=1e-13, offset
         # first calculate norm
         initial_norm = A[i].norm(product)[0]
 
-        if initial_norm < atol:
+        if initial_norm <= atol:
```

Comment:
Maybe leave a code comment why `<=` is required?

Reply:
I decided to revert this change and use `gram_schmidt` in `qr_svd` with default tolerances. That way, using `atol=0` and `rtol=0` guarantees no vector will be removed.